src/hooks/useFriction.ts

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { CSSPairs, getInterpolatorForPair } from '../helpers/pairs';
+import { Friction1DParams, friction1D } from '../animation/friction';
+
+type UseFrictionArgs = CSSPairs &
+  Omit<Friction1DParams, 'onUpdate' | 'onComplete'>;
+
+export const useFriction = <M extends HTMLElement>({
```

Comment:
In truly great fashion, our `useFriction` hook is nearly _identical_ to our `useGravity` hook, with just a few different config parameters and calling of a different physics calculation (`friction1D` vs. `gravity1D`). 🎉 Love when it's this easy!